Set sane defaults for the root certificate stores in MacPorts

This patch just adds placeholders; the correct paths will be added by the
Portfile, because they are installation-specific and depend on ${prefix}. The
file can either be provided by curl-ca-bundle maintained with curl, or be
generated from the system keychain using certsync.

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Clemens Lang <cal@macports.org>
--- ./doc/Muttrc.head.orig	2014-03-12 17:03:44.000000000 +0100
+++ ./doc/Muttrc.head	2014-03-12 17:03:44.000000000 +0100
@@ -97,6 +97,12 @@
 attachments   -I message/external-body
 
 ##
+## MacPorts-specific configuration to seed the trust store
+##
+set ssl_ca_certificates_file="@CA_PATH@"
+set smime_ca_location="@CA_PATH@"
+
+##
 ## More settings
 ##
 
